src/GovUk.Education.ExploreEducationStatistics.Admin/Services/DataBlockService.cs

```diff
@@ -329,6 +331,15 @@ private List<DataBlock> GetDataBlocks(Guid releaseId, Guid? subjectId = null)
                 .OnSuccessVoid();
         }
 
+        public async Task InvalidateCachedDataBlocks(Guid releaseId)
+        {
+            var dataBlocks = GetDataBlocks(releaseId);
+            foreach (var dataBlock in dataBlocks)
+            {
+                 await InvalidateDataBlockCache(releaseId, dataBlock.Id);
```

Comment:
Would be tempted to rename this to `InvalidateCachedDataBlock` for consistency with our new method